public/app/features/provisioning/SyncRepository.tsx

```diff
@@ -20,53 +19,49 @@ export function SyncRepository({ repository }: Props) {
   const [isModalOpen, setIsModalOpen] = useState(false);
   const navigate = useNavigate();
   const name = repository.metadata?.name;
-  const folder = repository.spec?.folder;
-
-  // TODO generate endpoints for this
-  const { value } = useAsync(async () => {
-    return getBackendSrv().get(`/apis/folder.grafana.app/v0alpha1/namespaces/${config.namespace}/folders/${folder}`);
-  });
 
   useEffect(() => {
-    // TODO... not true anymore --
-    // clicking sync just means we queued the job; can watch it until finished
     const appEvents = getAppEvents();
     if (syncQuery.isSuccess) {
       appEvents.publish({
         type: AppEvents.alertSuccess.name,
-        payload: ['Resources imported'],
+        payload: ['Sync started'],
       });
-      // TODO the URL will be different based on the type of resource imported
-      navigate(`/dashboards${folder ? `/f/${folder}` : ''}`);
```

Comment:
The redirect here was a bit confusing -- now that we show a spinner I think this feels better

Reply:
thank ~god~ ryan!